inputfiles/overridingTypes.json

```diff
@@ -3096,7 +3092,7 @@
     "typedefs": {
         "typedef": [
             {
-                "override-type": "Blob | BufferSource | FormData | URLSearchParams | ReadableStream<Uint8Array> | string",
+                "override-type": "ReadableStream<Uint8Array> | XMLHttpRequestBodyInit",
```

Comment:
`BodyInit` is now [defined](https://fetch.spec.whatwg.org/#bodyinit-unions) as:
```
typedef (Blob or BufferSource or FormData or URLSearchParams or USVString) XMLHttpRequestBodyInit;

typedef (ReadableStream or XMLHttpRequestBodyInit) BodyInit;
```
This change re-aligns our `override-type` with the new definition.